src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -236,6 +236,14 @@ bool layout_optimizer::can_fuse_reorder(program_node& prev, program_node& next,
     if (next.is_type<reorder>())
         return true;
 
+    // Errata for onednn layout selection
+    if (next.is_type<convolution>() &&
+        next.get_preferred_impl_type() == impl_types::onednn &&
+        ((fmt_prev == format::byxf && fmt_next == format::byxf) ||
+         (fmt_prev == format::bfyx && fmt_next == format::byxf))) {
+        return true;
```

Comment:
Could you check performance on gen9 and gen12lp?

Reply:
http://ikswven004.ikor.intel.com:8080/job/release-i7-8700k-avagen9-05-dlbenchmark/3/
http://ikswven004.ikor.intel.com:8080/job/release-i7-1165G7-avagen12-1-dlbenchmark/7/